packages/atomic/src/components/search/tabs/atomic-tab-section/atomic-tab-section.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  loadBreadcrumbActions,
+  loadQueryActions,
+  loadSearchActions,
+  loadSearchAnalyticsActions,
+  loadTabSetActions,
+} from '@coveo/headless';
+import {
+  Component,
+  h,
+  Element,
+  State,
+  Listen,
+  Prop,
+  EventEmitter,
+  Event,
+} from '@stencil/core';
+import ArrowDown from '../../../../images/arrow-down.svg';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {getActiveTab} from '../../../../utils/tab-utils';
+import {Button} from '../../../common/button';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-tab-section',
+  styleUrl: 'atomic-tab-section.pcss',
+  shadow: true,
+})
+export class AtomicTabSection {
+  @InitializeBindings() public bindings!: Bindings;
+  @Element()
+  private host!: HTMLElement;
+
+  /**
+   * When provided, this tab will be selected by default when the component loads. Otherwise, the first tab is selected automatically.
```

Comment:
@fbeaudoincoveo This had me realize that instead of setting a default tab in atomic-tab-section (and having to trigger a hash update in atomic-search-interface so that the url reflects the current state), we can set a default tab by setting the active prop on the desired tab. 

This seems a lot simpler to me, but what do you think?

Reply:
I think the simplest and best approach would be to try to get the active tab from the URL state and fallback to the first tab found under the (soon to be renamed?) `atomic-tab-section` component. This should cover most use cases. After all, why would you want to select anything but the first tab by default?

One problem I see with having an `active` prop on each atomic-tab component is that you have to deal with possible misconfigurations. E.g., what happens when someone sets `active` to `true` on multiple tabs? You'd probably want to throw a warning or something that explains what tab was selected due to the misconfiguration... It feels a bit awkward.

Arguably there's a similar concern with having a prop on the `atomic-tab-section`: what if the specified tab ID doesn't exist? Do you throw a runtime error or gracefully select the first tab as a fallback and show a console warning?...

I'd get rid of both props 😁

Reply:
I guess we can do that, if someones wants to set another tab by default, they can just change the link to the search page..!

The only downside I see is that if we want the url to have the correct tab name on initial load without triggering another search we need to update it.

I had added an event listener in atomic-search-interface for an event triggered by atomic-search-manager that would call updateHash(), but do you know a better way I could do this?

Reply:
Actually, it does not matter if we don't show the tab name in the url on first load, it will always be the first one anyway.